docs/content/manual/pre-release/node/kubelet-restart-on-a-node.md

```diff
@@ -11,34 +11,36 @@ https://github.com/longhorn/longhorn/issues/2629
 3. Deploy prometheus monitoring app on the cluster which is using Longhorn storage class or deploy a statefulSet with Longhorn volume.
 4. Write some data into the mount point and compute the md5sum.
 5. Restart the kubelet on the node where the statefulSet or Prometheus pod is running using the command `sudo docker restart kubelet`
-6. Observe the volume. It gets detached and the pod gets terminated.
-7. Once the pod is terminated, a new pod should be created and get attached to the volume successfully.
-8. Verify that the mount of the volume is successful and data is safe.
+6. Observe the volume. It becomes degraded but is still running. 
```

Comment:
Just wanted to understand here, after restarting the kubelet on the node where the pod/volume is attached, will the engine not restart and result in volume detaching and attaching again?

Reply:
No. The volume won't become detached/crashed. Instead, it will keep running.

Reply:
Great!